src/App.js

```diff
@@ -1,25 +1,71 @@
 import './App.css';
-import { Routes, Route, Link } from "react-router-dom";
-import { DynamicItem, Sidebar, SideBarMenuItems } from "./components";
-
+import React, { useEffect, useState, useMemo } from 'react';
+import { Routes, Route, Navigate } from 'react-router-dom';
+import { AuthContext } from './context/AuthContext';
+import { DynamicItem, Sidebar, SideBarMenuItems } from './components';
+import Login from './pages/Login';
+import SignUp from './pages/SignUp';
+import TeacherDetail from './components/Teacher/Teacher-Detail';
 
 function App() {
+  const [user, setUser] = useState(null);
```

Comment:
- [ ] Your app has some Eslint errors and it's refusing to compile with `npm run start` or `npm start`.  Kindly fix those linter errors so your app can run as intended. 👍 

- [ ] All over your frontend project, you are using the "function" keyword.  Please, refactor those functions to ES6 arrow functions.

## Explanation

All over the software industry, the "function" keyword has been soft deprecated in favor of ES6 functions.

## Tips and extra help

Just do a quick search in Vscode for the word "function" so you can find them all, and change each one to ES6 arrow functions.  Don't worry, it should take no more than 5-10 minutes.  

That way,  your app will comply with latest standards 👍